run-spread

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# check dependencies
+if ! snap list snapcraft &>/dev/null; then
+    echo "snapcraft is not installed"
+    exit 1
+fi
+
+# Clean the snaps created in previous runs
+touch test-build
+rm -rf built-snap
+mkdir built-snap
+rm -f snapd_1337.*.snap snapd_1337.*.snap.keep
+
+# Build snapd snap
+snapcraft --use-lxd
+for snap in built-snap/snapd_1337.*.snap; do
+    mv "${snap}" "${snap}.keep"
+done
+
+# Run spread
+spread "$@"
```

Comment:
How does `spread` know to use the snaps ending in `.keep` instead of rebuilding them from scratch?

Reply:
the code is in build_snapd_snap function in prepare.sh
we use .keep to avoid .gitignore policy